### PR TITLE
[sw]: Translation of added keywords

### PR DESCRIPTION
**Pull request by Pheogrammer**
Swahili translation of added keywords

### CODE REVIEW

File: locales/sw/json.json [hunk 1]
```diff
@@ -77,13 +77,13 @@
...
     "April": "Aprili",
-    "Are you sure you want to attempt to pay :amount?": "Are you sure you want to attempt to pay :amount?",
+    "Are you sure you want to attempt to pay :amount?": "Je, una uhakika unataka kujaribu kulipa :amount?",
     "Are you sure you want to cancel your subscription?": "Je, una uhakika unataka kughairi usajili wako?",
     "Are you sure you want to delete the selected resources?": "Je, una uhakika unataka kufuta rasilimali zilizochaguliwa?",
     "Are you sure you want to delete this announcement?": "Je, una uhakika unataka kufuta tangazo hili?",
     "Are you sure you want to delete this file?": "Je, una uhakika unataka kufuta faili hii?",
     "Are you sure you want to delete this notification?": "Je, una uhakika unataka kufuta arifa hii?",
-    "Are you sure you want to delete this payment method?": "Are you sure you want to delete this payment method?",
+    "Are you sure you want to delete this payment method?": "Je, una uhakika unataka kufuta njia hii ya kulipa",
```
Comment (andrey-helldar) [inline locales/sw/json.json:86]: Surely there shouldn't be a question mark here?

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/sw/json.json (hunk 1 above)
Comment (Pheogrammer) [reply locales/sw/json.json]: Oh, I have fixed it

**Review by andrey-helldar — APPROVED** [review]
Ahsante!

**Comment by andrey-helldar** [comment]
You can run the `composer update` console command. Fix was released 🙂

**Pull request merged**